frontend/src/routes.tsx

```diff
@@ -63,39 +64,39 @@ const defaultRoute = '/overview';
 const pathRoutes: Path[] = [
   {
     path: '/overview',
-    component: OverviewPageContainer
+    component: OverviewPage
   },
   {
     path: '/graph/node/namespaces/:namespace/' + Paths.AGGREGATES + '/:aggregate/:aggregateValue',
-    component: GraphPageContainer
+    component: GraphRoute
```

Comment:
@ferhoyos I know this is already merged, but what I don't understand is how Redux is still working when neither the routes nor the wrappers are referring to the redux containers but rather the react component directly.   Here the component is changed from GraphPageContainer to GraphRoute, but in GraphRoute I see `<GraphPage>` not `<GraphPageContainer>`, for example.

Reply:
When you import a default export you can set whatever name you want, it is not mandatory to use the same name of the export (because there is only one default export per file). Only with normal export you have to import the same variable name (because there can be many of them per file).

An example:
```
export default GraphPageContainer;
import GraphPage from 'xxx/GraphPage'  //correct 
import AAA from 'xxx/GraphPage'  //correct 
```
```
export GraphPageContainer;
import { GraphPage } from 'xxx/GraphPage' //wrong
import { GraphPageContainer } from 'xxx/GraphPage' //correct
```

There are other examples in Kiali of different import names with default exports:
```
reducers/TourState:
export default tour;

reducers/index:
import TourState from './TourState';
```

Reply:
@ferhoyos Thanks for that explanation!  I was confused.  And now I know why, it's confusing!   I totally agree, we should remove the use of default export, I don't think there is a huge benefit to being able to rename the component being exported by default, and it can lead to confusion.